Sources/ApolloWebSocket/WebSocketError.swift

```diff
@@ -24,6 +25,8 @@ public struct WebSocketError: Error, LocalizedError {
         return "Websocket error: Serialized message not found"
       case .neitherErrorNorPayloadReceived:
         return "Websocket error: Did not receive an error or a payload."
+      case .upgradeError:
+        return "Websocket error: Invalid HTTP upgrade."
```

Comment:
Since we've got the code as part of the error is it worth at least putting the error code in this message?

Reply:
Same could really be said for all the other errors too though - "Invalid HTTP upgrade" is quite specific. Not sold that the additional code is needed.

Reply:
fair enough